frontend/src/scenes/me/Settings/ChangePassword.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { toast } from 'react-toastify'
+import api from 'lib/api'
+import { Input, Button } from 'antd'
+import { useActions, useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import Form from 'antd/lib/form/Form'
+import FormItem from 'antd/lib/form/FormItem'
+
+export function ChangePassword(): JSX.Element {
+    const { user } = useValues(userLogic)
+    const { loadUser } = useActions(userLogic)
+
+    const [currentPassword, setCurrentPassword] = useState('')
+    const [newPassword, setNewPassword] = useState('')
+
+    async function submit(): Promise<void> {
+        try {
+            await api.update('api/user/change_password', {
```

Comment:
Not sure if you want to convert this to kea logic in this opportunity, or leave as an improvement for later